### PR TITLE
📖 springAds: update md file

### PR DESCRIPTION
**Pull request by renebaudisch**
update md file

### CODE REVIEW

File: ads/springAds.md [hunk 1]
```diff
@@ -44,18 +44,15 @@ limitations under the License.
 >
 </amp-ad>
 <amp-ad width="320" height="50" type="springAds" data-adslot="banner"></amp-ad>
-<amp-ad width="300" height="250" type="springAds" data-adslot="mrec"></amp-ad>
+<amp-ad width="100vW" height="fluid" type="springAds" data-adslot="mrec"></amp-ad>
```
Comment (lannka) [inline ads/springAds.md:47]: `100vw`

could you also update your example in ads.amp.html and make sure it's working?

**Comment by lannka** [comment]
@renebaudisch pls take a look at the travis error logs and fix the code format

**Comment by mrjoro** [comment]
Here's the info from the failed prettify check:

```
[08:36:04] Suggested fixes for ads/springAds.md:
@@ -45,10 +45,5 @@ limitations under the License.
-<amp-ad 
-  width="320" 
-  height="50" 
-  type="springAds" 
-  data-adslot="banner"
-></amp-ad>
-<amp-ad 
-  width="100vW" 
-  height="fluid" 
-  type="springAds" 
+<amp-ad width="320" height="50" type="springAds" data-adslot="banner"></amp-ad>
+<amp-ad
+  width="100vW"
+  height="fluid"
+  type="springAds"
@@ -57,0 +53 @@ limitations under the License.
+
[08:36:04] NOTE 1: If you are using GitHub's web-UI to edit files, copy the suggested fixes printed above into your PR.
[08:36:04] NOTE 2: If you are using the git command-line workflow, run gulp prettify --local_changes --fix from your local branch.
[08:36:04] NOTE 3: Since this is a destructive operation (that edits your files in-place), make sure you commit before running the command.
[08:36:04] NOTE 4: For more information, read https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#code-quality-and-style
```

File: ads/springAds.md [hunk 2]
```diff
@@ -43,19 +42,26 @@ limitations under the License.
...
 >
 </amp-ad>
-<amp-ad width="320" height="50" type="springAds" data-adslot="banner"></amp-ad>
-<amp-ad width="300" height="250" type="springAds" data-adslot="mrec"></amp-ad>
+<amp-ad 
+  width="320" 
+  height="50" 
+  type="springAds" 
+  data-adslot="banner"
+></amp-ad>
```
Comment (mrjoro) [inline ads/springAds.md:50]:
```suggestion
<amp-ad width="320" height="50" type="springAds" data-adslot="banner"></amp-ad>
```

File: ads/springAds.md [hunk 3]
```diff
@@ -43,19 +42,26 @@ limitations under the License.
...
+<amp-ad 
+  width="100vW" 
+  height="fluid" 
+  type="springAds" 
+  data-adslot="mrec"
+></amp-ad>
```
Comment (mrjoro) [inline ads/springAds.md:56]:
```suggestion
<amp-ad
  width="100vW"
  height="fluid"
  type="springAds"
  data-adslot="mrec"
></amp-ad>
```

File: ads/springAds.md [hunk 4]
```diff
@@ -43,19 +42,26 @@ limitations under the License.
...
 ```
+For a maintenanced uptodate documentation please refer to our
```
Comment (mrjoro) [inline ads/springAds.md:58]:
```suggestion

For a maintenanced uptodate documentation please refer to our
```

**Pull request merged**